### DISCUSSION TITLE
How to switch Keycloak's development mode to production mode

### DISCUSSION THREAD

**Discussion by muzafferjoya** [keycloak/keycloak#25803] [Q&A]
I have added my docker compose file here for reference

`  keycloak:
    image: quay.io/keycloak/keycloak:20.0.0
    container_name: keycloak
    command:
      [
        "start-dev",
        "--http-port=8081",
        "--http-relative-path=/auth"
      ]
    ports:
      - "8081:8081"
    environment:
      - KEYCLOAK_ADMIN=admin
      - KEYCLOAK_ADMIN_PASSWORD=admin1234
      - KC_DB=postgres
      - KC_DB_URL=jdbc:postgresql://postgresdb:5432/keycloak
      - KC_DB_USERNAME=postgres
      - KC_DB_PASSWORD=postgres
`

**Comment by vmuzikar** [comment]
Instead of `start-dev` command you can you `start`.
